src/WebJobs.Script.WebHost/Diagnostics/BufferedConsoleWriter.cs

```diff
@@ -108,12 +111,12 @@ private async Task ProcessConsoleBufferAsync()
                             }
 
                             _writeResetEvent.Set();
-                            Console.Write(builder.ToString());
+                            Writer.Write(builder.ToString());
```

Comment:
Should we consider using the [async version](https://learn.microsoft.com/en-us/dotnet/api/system.io.textwriter.writeasync?view=net-8.0) of this method?

Reply:
Same q for the other usages and the WriteLine method below at 119

Reply:
Sure, will make async.